tensorflow/python/keras/utils/vis_utils.py

```diff
@@ -124,6 +146,27 @@ def model_to_dot(model,
     dot.set('dpi', dpi)
     dot.set_node_defaults(shape='record')
 
+  if layer_range is None:
+    layer_range = [
+      model.get_layer(index=0).name,
```

Comment:
I don't think its necessary to populate this with first and last layer names, can we just ignore layer_range if it is none?

Reply:
done